### PR TITLE
submitted an initial solution to week 6

### PR DESCRIPTION
**Pull request by Andrey8860**

### CODE REVIEW

File: Week 6/solution/pom.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	<name>icanwin</name>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.source>1.7</maven.compiler.source>
+		<maven.compiler.target>1.7</maven.compiler.target>
+	</properties>
+
+	<dependencies>
+		<dependency>
```
Comment (mamax) [inline Week 6/solution/pom.xml:19]: Why do you use TestNG and JUnit simultaneously? need to use one of thosee libraries, but not both

File: Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package com.epam.week6.tests;
+
+import java.time.Duration;
+import org.junit.Assert;
```
Comment (mamax) [inline Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java:4]: Do not use assertions from JUnit
`import org.testng.Assert;`

File: Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+		String loginEmail = "testreg.12.4.2022_1@mailinator.com";
+		String loginPassword = "Testtest&1";
+		
+		WebDriver driver = new ChromeDriver();
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+		driver.manage().window().maximize();
+		
+		driver.get("https://qa3-www.sportchek.ca/en.html");
+        ((WebStorage) driver).getLocalStorage().setItem("preferred-store-id", "371");
+		driver.findElement(By.cssSelector(".nl-sm-block .nl-pencil-banner__login")).click();
```
Comment (mamax) [inline Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java:26]: Locators should be declared inside page classes.

File: Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class SportcheckSignOutWihoutPageObjectModelTest {
+	
+	@Test
+	public void sportcheckLoginToastMessageTest() {
+		String loginEmail = "testreg.12.4.2022_1@mailinator.com";
+		String loginPassword = "Testtest&1";
+
```
Comment (mamax) [inline Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java:19]: The path to the driver executable must be set by the `webdriver.chrome.driver` system property
Please use [https://github.com/bonigarcia/webdrivermanager](https://github.com/bonigarcia/webdrivermanager) librsry

File: Week 6/solution/pom.xml [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+	<groupId>com.epam.week6</groupId>
+	<artifactId>icanwin</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+
+	<name>icanwin</name>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.source>1.7</maven.compiler.source>
```
[the remark this answers is not in the document]
Comment (Andrey8860) [reply Week 6/solution/pom.xml]: Resolved

File: Week 6/solution/src/test/java/com/epam/week6/tests/BaseTest.java [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+	protected WebDriver driver;
+	
+	@BeforeMethod(alwaysRun = true)
+	public void setUp() {
+		this.driver = new ChromeDriver();
+	}
+	
+	@AfterMethod(alwaysRun = true)
+	public void tearDown() {
+		driver.quit();
```
[the remark this answers is not in the document]
Comment (Andrey8860) [reply Week 6/solution/src/test/java/com/epam/week6/tests/BaseTest.java]: Resolved

File: Week 6/solution/src/test/java/com/epam/week6/tests/GoogleCloudCalculatorTests.java [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+		googleCloudPricingCalculatorPage.clickSendEmailButtonOnEmailYourEstimateForm();
+		
+		driver.switchTo().window(windowHandles.get(1));
+		
+		// Here the do-while loop is used, since we need to click the refresh button at 
+		// least once to see if email has come, and if it is not here yet - 
+		// we need to continue clicking
+		do {
+			yopmailEmailInboxPage.clickRefreshEmailsButton();
+		} while(driver.findElements(By.xpath("//div[text()='1 mail']")).size() == 0);
```
[the remark this answers is not in the document]
Comment (Andrey8860) [reply Week 6/solution/src/test/java/com/epam/week6/tests/GoogleCloudCalculatorTests.java]: Resolved

File: Week 6/solution/pom.xml (hunk 1 above)
Comment (Andrey8860) [reply Week 6/solution/pom.xml]: Copy/paste mistake. Resolved

File: Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java (hunk 2 above)
Comment (Andrey8860) [reply Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java]: Resolved

File: Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java (hunk 3 above)
Comment (Andrey8860) [reply Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java:26]: This particular class was requested in the homepage to be without any POM model. It is reworked with a POM model in week 7. While it is strange - I did as requested. That is why there is no structure - this test is all by itself.

File: Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java (hunk 4 above)
Comment (Andrey8860) [reply Week 6/solution/src/test/java/com/epam/week6/tests/SportcheckSignOutWihoutPageObjectModelTest.java:19]: This library was first covered in week 9, as it turned out. I added it retroactively to weeks 6-8 as well. An exception is [SportcheckSignOutWihoutPageObjectModelTest.java] class, due to the reason mentioned in the previous comment

**Pull request merged**